lib/plain/entities/concept-scheme.ts

```diff
@@ -51,10 +51,13 @@ export type ConceptSchemePlainClientAPI = {
    * @returns the updated Concept Scheme
    * @throws if the request fails
    * @see {@link https://www.contentful.com/developers/docs/references/content-management-api/#/reference/taxonomy/concept-scheme}
+   * @deprecated The behavior of this method as a PATCH is being deprecated, and will be replaced with a PUT in the next major version. Use the `patch` method instead.
```

Comment:
> in the next major version

how can we make sure we don't forget to update it?

Reply:
btw the last time we updated the major version was more than a year ago. long process

Reply:
Fair point, but the impression I get from Tundra from our slack thread is that they'll work with us on this as planning for the next version.

Reply:
what just if...

```ts
import * as package from 'package.json'

it('should not pass if major version is updated', () => {
  if (package.version.split('.')[0] === 12) {
    expect('updatePut' in sdk).to.be(false)
  }
})
```